src/sentry/lang/native/sources.py

```diff
@@ -269,6 +269,7 @@ def get_internal_artifact_lookup_source_url(project: Project):
 def get_bundle_index_urls(
     project: Project, release: str | None, dist: str | None
 ) -> tuple[str | None, str | None]:
+    # TODO this random rollout is backwards
```

Comment:
this is intentional, as its an early return.
also NOTE to self: this feature ultimately didn’t ship, and I should work towards removing it at this point.